test/system/navbar_links_test.rb

```diff
@@ -85,7 +85,7 @@ class NavbarLinksTest < ApplicationSystemTestCase
       end
 
       it 'should not display an admin link' do
-        refute has_link? t('navigation.admin_tools.label')
+        refute has_button? t('navigation.admin_tools.label')
```

Comment:
Since matchers like 'has_button' return after either the matcher is satisfied OR the default wait time is reached, this anti-pattern may be adding 5 seconds to each test of this form.This isn't a blocker thought so I can spin off another PR for it if you'd like

Reply:
The matcher waits are intentional here - poltergeist used to be pretty spotty about this kind of thing, and I still don't completely trust the new system test regime completely. You are right that this could slow build times down, but I'm less convinced that long build times are a pain point we're experiencing at the moment. Do you feel differently, or is there a different issue worth raising?